lib/blockchain/cardano.js

```diff
@@ -0,0 +1,47 @@
+const fs = require('fs')
+const path = require('path')
+const common = require('./common')
+const mnemonicHelpers = require('../../lib/mnemonic-helpers')
+
+const coinUtils = require('../coin-utils')
+const options = require('../options')
+
+module.exports = { setup }
+
+const es = common.es
+
+// todo: maybe add '#' as second parameter?
+function escapeRegExp(str) {
+  return str.replace(/[.*+?^${}()|[\]\\#]/g, '\\$&')
+}
+
+function setup(dataDir) {
+  const coinRec = coinUtils.getCryptoCurrency('ADA')
+
+  // todo: maybe use template tag for escaping?
+  es(
```

Comment:
ok

Reply:
An alternative -- can't we symlink `state-wallet-testnet` instead of rewriting the config?
Also, don't we have an "autoescaping" version of the shell launcher? Something like python's
`os.system(['sudo', 'sed', '-i', '.....', '/usr/...']) ?

Reply:
`state-wallet-testnet is symlink` to something like /nix/5435345/353.... - so i really copy this originall file - don't forget that we compile that build in /tmp folder...

but what i don't understand - where is advantage of that symlink? you still need to rewrite that datadir paths... 

yes, they have spawn method, i try to use this.